debezium-server/debezium-server-redis/src/main/java/io/debezium/server/redis/RedisStreamChangeConsumer.java

```diff
@@ -69,11 +70,11 @@ public class RedisStreamChangeConsumer extends BaseChangeConsumer
     Integer batchSize;
 
     @ConfigProperty(name = PROP_PREFIX + "retry.initial.delay.ms", defaultValue = "300")
-    Integer initialRetryDelay;
-
+    Integer initialRetryDelay1;
+    Duration initialRetryDelay = Duration.ofMillis(initialRetryDelay1);
```

Comment:
I'd keep the injection points here as they are and just inline the conversion in `handleBatch` method.